activiti-cloud-query-service/activiti-cloud-starter-query/src/main/resources/config/query/liquibase/master.xml

```diff
@@ -99,5 +99,25 @@
         splitStatements="true"
         stripComments="true"/>
   </changeSet>
+
+  <changeSet author="activiti-query"
+      id="alter5-schema-m9" dbms="postgresql">
+    <sqlFile dbms="postgresql"
+        encoding="utf8"
+        path="changelog/06-alter.pg.schema.7.1.0-m9.sql"
```

Comment:
What is the right name to use here?

Reply:
The name sounds ok. The only point is that you need to be using `m10` suffix instead of `m9` as we are currently in the milestone 10. Also, this [another PR](https://github.com/Activiti/activiti-cloud/pull/194/files#diff-9644d6bb9ccce725c739db2cfcdd56f0R1) is also modifying the database it will have a conflicting name with yours, so better to go with `changelog/07-alter.pg.schema.7.1.0-m10.sql`